src/Nethermind/Nethermind.JsonRpc/Modules/Trace/ITraceRpcModule.cs

```diff
@@ -27,7 +28,7 @@ public interface ITraceRpcModule : IRpcModule
         ResultWrapper<ParityTxTraceFromReplay> trace_call(TransactionForRpc message, string[] traceTypes, BlockParameter blockParameter);
         
         [JsonRpcMethod(Description = "", IsImplemented = false, IsSharable = false)]
-        ResultWrapper<ParityTxTraceFromReplay[]> trace_callMany((TransactionForRpc message, string[] traceTypes, BlockParameter numberOrTag)[] calls);
+        ResultWrapper<ParityTxTraceFromReplay[]> trace_callMany(Tuple<TransactionForRpc, string[]>[] calls, BlockParameter numberOrTag);
```

Comment:
Why Tuple and not class? Might save us issue with deserialization.

Reply:
Changed.